pkgs/by-name/bu/buildstream/package.nix

```
@@ -126,6 +133,12 @@ python3Packages.buildPythonApplication rec {
     "tests/internals/cascache.py"
   ];
 
+  postInstall = ''
+    installShellCompletion --cmd bst \
+      --bash $src/src/buildstream/data/bst \
+      --zsh $src/src/buildstream/data/zsh/_bst
```

```suggestion
      --bash src/buildstream/data/bst \
      --zsh src/buildstream/data/zsh/_bst
```
we do not want to omit patches otherwise LGTM